src/javascript/app/base/subscription_manager.js

```diff
@@ -0,0 +1,204 @@
+import BinarySocket     from './socket';
```

Comment:
@aaron-binary 
It'd be better to move the `subscribtion_manager.js` from `src/javascript/app_2/Services/` to `src/javascript/_common/` and use it in both projects instead of duplicating it.